fix(acl): add the placeholders manual for build-in acl file (#640)
emqtt/emq-docs-cn: en_US/advanced/acl-file.md

The rules file is described in Erlang syntax:

```erlang

> %% Allow "dashboard" users to subscribe to "$ SYS/#" topics

Reviewer: why the space after `$`?

Author: Nice eyes!

    * `subscribe`：The rule applies to SUBSCRIBE operations
    * `pubsub`：The rule applies to both PUBLISH and SUBSCRIBE operations
- The fourth position of the tuple means the list of topics restricted by the rule. The content is given in the form of an array. For example:

>     * `"$SYS/#"`：a **Topic Filter** which means that the rule can hit topics that match `$SYS/#`; for example, it can hit "$SYS/#" and "$SYS/a/b/c"

Reviewer: ```suggestion
    * `"$SYS/#"`：a **Topic Filter** which means that the rule is applied to topics that match `$SYS/#`; for example rules created for "$SYS/#" applies to publish/subscribe actions for topic "$SYS/a/b/c"
```

>     * `{eq, "#"}`：It indicates full equivalence of characters. The rule can only hit strings with the topic `#`. It cannot hit `/a/b/c`, etc.

Reviewer: ```suggestion
    * `{eq, "#"}`：It indicates full equivalence of characters. The rule is only applied for topic `#` but not for `/a/b/c`, etc.
```

After the `acl.conf` modification is completed, it will not be automatically loaded into the EMQ X Broker system, but needs to be performed manually:

```bash
./bin/emqx_ctl modules reload emqx_mod_acl_internal
```

## Placeholders

The built-in `acl.conf` supports only the following placeholders in the subject's field (the 4th position of the tuple).

> - `%c`: Indicates the Client ID, which will be replaced with the actual client ID when the rule takes effect.

Reviewer: ```suggestion
- `%c`: For Client ID, which is replaced by the client ID when the rule takes effect.
```

> - `%u`: Indicates the client username, which will be replaced with the actual client username when the rule takes effect.

Reviewer: ```suggestion
- `%u`: For username, which is replaced by the client's username when the rule takes effect.
```

E.g:

```erlang
{allow, all, pubsub, ["sensor/%c/ctrl"]}.
```

> Means that clients with ID light are **Allowed** to **Subscribe and Publish** to the `sensor/light/ctrl` topic.

Reviewer: ```suggestion
Means that a client with ID 'light' is **Allowed** to **Subscribe and Publish** to the `sensor/light/ctrl` topic.
```
